tests/test_search_itp_lithologischebeschrijvingen.py

```diff
@@ -179,8 +179,6 @@ def test_search_customreturnfields(self, mp_remote_describefeaturetype,
         assert list(df) == ['pkey_interpretatie', 'pkey_boring',
                             ]
 
-        assert not pd.isnull(df.pkey_boring[0])
```

Comment:
This should not be removed; it is a valid test since all 'lithologische beschrijvingen' interpretations are linked to boreholes, thus pkey_boring should always be present and not null.

The reason this test failed is because there is an extra line in types/interpretations that sets pkey_boring to NaN erroneously.

Reply:
Thx Roel, good remark about changing 'custom' to 'wfs' if only pkey_boring is considered